### PR TITLE
[CF-161] VerifyVms shows incorrect results

### PR DESCRIPTION
**Pull request by mirrorcoder**
adding in failed_vms if only vm don't found in dst

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by miarmak** [comment]
+1

**Comment by SVilgelm** [comment]
:-1: This is not solving the problem, just muting, we have to have an understanding what does correctly migrated instance mean.

**Comment by miarmak** [comment]
I guess **correctly migrated instance** is that instance, that was migrated (recreated) on destination as is. But we have so many cases, when it is not done in this way (recreated flavor or image can differ from source for example by id), but still VM is migrated as was supposed. So IMO in this action that would be fine if we fix it in this way.

File: cloudferrylib/os/actions/verify_vms.py [hunk 1]
```diff
@@ -70,25 +70,20 @@ def run(self, **kwargs):
...
-                    failed_vms.append(src_inst_id)
                 if (src_inst_info['flav_details'] !=
                         dst_cmp_inst['flav_details']):
                     LOG.warning("Wrong flav_details of instance %s on DST",
                                 src_inst_id)
-                    failed_vms.append(src_inst_id)
                 if src_inst_info['key_name'] != dst_cmp_inst['key_name']:
                     LOG.warning("Wrong key_name of instance %s on DST",
                                 src_inst_id)
-                    failed_vms.append(src_inst_id)
```
Comment (roman-verchikov) [inline cloudferrylib/os/actions/verify_vms.py]: Key name must always stay the same

File: cloudferrylib/os/actions/verify_vms.py [hunk 2]
```diff
@@ -70,25 +70,20 @@ def run(self, **kwargs):
                 if src_inst_info['name'] != dst_cmp_inst['name']:
                     LOG.warning("Wrong name of instance %s on DST",
                                 src_inst_id)
-                    failed_vms.append(src_inst_id)
```
Comment (roman-verchikov) [inline cloudferrylib/os/actions/verify_vms.py]: VM name must always stay the same

File: cloudferrylib/os/actions/verify_vms.py [hunk 3]
```diff
@@ -70,25 +70,20 @@ def run(self, **kwargs):
...
                 if src_inst_info['key_name'] != dst_cmp_inst['key_name']:
                     LOG.warning("Wrong key_name of instance %s on DST",
                                 src_inst_id)
-                    failed_vms.append(src_inst_id)
                 if (sorted(src_inst_info['interfaces']) !=
                         sorted(dst_cmp_inst['interfaces'])):
                     LOG.warning("Wrong interfaces of instance %s on DST",
                                 src_inst_id)
-                    failed_vms.append(src_inst_id)
```
Comment (roman-verchikov) [inline cloudferrylib/os/actions/verify_vms.py]: Interfaces must always stay the same, it's one of the most critical points to check and defines migration success more than all other attributes combined.

**Comment by mirrorcoder** [comment]
fix

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
can be merged

**Pull request merged**